examples/amp-story/amp-story-shopping.html

```diff
@@ -17,20 +17,122 @@
         publisher="AMP Story Shopping"
         publisher-logo-src="example.com/logo.png"
         poster-portrait-src="example.com/poster.jpg">
-      <amp-story-page id="default">
-        <amp-story-shopping-config layout='nodisplay'></amp-story-shopping-config>
+      <amp-story-page id="src-example-inline-light-theme">
+        <!--
+          Inline Shopping Component Config
+        -->
+        <amp-story-shopping-config layout='nodisplay'>
+          <script type="application/json">
+            {
+                "items" : [
+                   {
+                    "product-tag-id": "hat",
+                    "brand-label": "...",
+                    "brand-favicon": "...",
+                    "product-title": "Hootenanny Hat",
+                    "product-price": "...",
+                    "product-images": "...",
+                    "product-details": "...",
+                    "reviews-page": "...",
+                    "reviews-data": "...",
+                    "cta-text": 1,
+                    "shipping-text": 1
+                   },
+                   {
+                    "product-tag-id": "sunglasses",
+                    "brand-label": "...",
+                    "brand-favicon": "...",
+                    "product-title": "Spectacular Spectacles",
+                    "product-price": "...",
+                    "product-images": "...",
+                    "product-details": "...",
+                    "reviews-page": "...",
+                    "reviews-data": "...",
+                    "cta-text": 1,
+                    "shipping-text": 1
+                   },
+                   {
+                    "product-tag-id": "backpack",
+                    "brand-label": "...",
+                    "brand-favicon": "...",
+                    "product-title": "Beastly Backpack",
+                    "product-price": "...",
+                    "product-images": "...",
+                    "product-details": "...",
+                    "reviews-page": "...",
+                    "reviews-data": "...",
+                    "cta-text": 1,
+                    "shipping-text": 1
+                   }
+                ]
+              }
+          </script>
+        </amp-story-shopping-config>
         <amp-story-grid-layer template="vertical">
-          <amp-story-shopping-tag></amp-story-shopping-tag>
+          <amp-story-shopping-tag tag-data-id="hat" ></amp-story-shopping-tag>
+          <amp-story-shopping-tag tag-data-id="sunglasses" ></amp-story-shopping-tag>
+          <amp-story-shopping-tag tag-data-id="backpack" ></amp-story-shopping-tag>
```

Comment:
We generally stay away from custom attributes and favor [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) in our code. 
Please change this attribute to `data-tag-id` to follow that convention. It can still be accessed with `setAttribute` and `setAttribute`.